awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.jsx

```diff
@@ -27,6 +27,32 @@ const StyledButton = styled(PFButton)`
     color: white;
   }
 `;
+const DataListItemCells = styled(PFDataListItemCells)`
```

Comment:
Do we need these on the other lists as well?

Reply:
@jakemcdermott Yeah we probably _could_ use those styling in other lists.  I could make them props that are passed into our `DatalistCell` component.  That way we have the choice to use them if we want to.